src/step-registry.js

```diff
@@ -42,11 +47,13 @@ StepRegistry.prototype.getStepTexts = function () {
   });
 };
 
-StepRegistry.prototype.getStepPositions = function(filePath) {
+StepRegistry.prototype.getStepPositions = function (filePath) {
   var stepPositions = [];
   for (var step in this.registry) {
-    if (this.registry[step].filePath === filePath){
-      stepPositions.push({stepValue : step, lineNumber: this.registry[step].line});
+    for (var i = 0; i < this.registry[step].fileLocations.length; i++) {
```

Comment:
Any specific reason for not using `forEach`?

Reply:
I tried foreach first. The jshint linter we are using kept on telling me that `using outside variables inside function scope will lead to confusing syntax`. I tried a few things. This looked cleaner to me.